Segment-ComScore/Classes/SEGComScoreIntegration.m

```diff
@@ -268,124 +272,78 @@ - (void)videoPlaybackStarted:(NSDictionary *)properties withOptions:(NSDictionar
 
     NSDictionary *map = @{
         @"ns_st_mp" : properties[@"video_player"] ?: @"*null",
+        @"ns_st_ci" : properties[@"content_asset_id"] ?: @"0"
     };
 
-    [self.streamAnalytics createPlaybackSessionWithLabels:map];
-
-    // The label ns_st_ci must be set through a setAsset call
-    [[self.streamAnalytics playbackSession] setAssetWithLabels:@{
-        @"ns_st_ci" : properties[@"content_asset_id"] ?: @"0"
-    }];
+    [self.streamAnalytics createPlaybackSession];
+    
+    SEGLog(@"[[SCORStreamingAnalytics streamAnalytics] createPlaybackSessionW]");
 
-    SEGLog(@"[[SCORStreamingAnalytics streamAnalytics] createPlaybackSessionWithLabels: %@]", map);
+    [self addMetaDataAssets:map withOptions:integrations];
```

Comment:
Don't we want to `addMetaDataAssets` before we call `[self.streamAnalytics createPlaybackSession];` on line 278?

Reply:
I don't think so. Because in documentation in [page 8 ](url)https://app.box.com/s/csd2t9sb9z0m6gdadg852ysl4724jac8/file/545661486639
the mentioned `[self.streamAnalytics createPlaybackSession]; `should be called first